app/helpers/embed_player_helper.rb

```diff
@@ -33,6 +36,22 @@ def embed_player_episode_url(ep, type = nil, preview = false)
     embed_params(params)
   end
 
+  def embed_player_podcast_url(podcast, options, preview = false)
+    params = {}
+    params[EMBED_PLAYER_FEED] = podcast.published_url
+
+    params[EMBED_PLAYER_PLAYLIST] = (options[:all_episodes] == "all") ? options[:all_episodes] : options[:episode_number]
+    params[EMBED_PLAYER_SEASON] = options[:season]
+    params[EMBED_PLAYER_CATEGORY] = options[:category]
+
+    # TODO: the height styling doesn't really work here in the way Play specifies how it needs to be for a playlist, so leaving this as a TODO for now.
```

Comment:
according to Play, the height styling for a playlist using the responsive card type is supposed to be 100% + 800 (or more)px, but i wasn't sure how to translate that in the iframe below, so i've commented this out for now.

Reply:
+1 I couldn't really figure that one out either.  Maybe @brandonhundt has some suggestions - but commenting out for now seems fine.

Reply:
the added styling in the wrapper div didn't quite work; the playlist doesn't list the full playlist if i uncomment the below.